src/ledger/LedgerTxnImpl.h

```diff
@@ -22,6 +22,19 @@ std::unordered_map<LedgerKey, std::shared_ptr<LedgerEntry const>>
 populateLoadedEntries(std::unordered_set<LedgerKey> const& keys,
                       std::vector<LedgerEntry> const& entries);
 
+struct AssetPair
+{
+    Asset buying;
```

Comment:
The naming convention here matches other plain-old-data structures used by the `LedgerTxn` family, such as `InflationWinner` and `LedgerTxnDelta`. I agree this doesn't match our typical naming convention, but I think it would be better to keep everything consistent. Happy to update it if you disagree though.

Reply:
yeah it's a bit annoying. I guess if we don't have methods, it's acceptable (the main reason for this naming convention is to avoid shadowing member variables with local variables)